README.md

```diff
@@ -43,45 +43,38 @@ this service will need to be run again and the tileserver restarted once the
 tilebuilder finishes (just run `docker-compose restart tileserver`).
 
 ## Make It Go
-In `docker-compose.yml`, the tileserver and rails database are listed as
-dependencies for the rails service. So to start the whole thing up (omitting
-tilebuilder, which only needs to run once) just run the following. Omit the
-`-d` flag if you prefer to see all of the rails server output.
+In `docker-compose.yml`, the tileserver and rails services are listed
+as dependencies for the nginx service. So to start the whole thing up
+(omitting tilebuilder, which only needs to run once) just run the following.
 ```
 $ docker-compose up -d nginx
 ```
 
-To spin all the services back down run the following.
+Once rails fully starts up, you can view the app running app at `localhost:3000`
+or an alternative port specified in `.env` if one exists. See `.env.example` for
+available options and reasonable starting values.
+
+Omit the `-d` flag if you prefer to see the nginx server output. You can always
+tail the output of any service with `docker-compose logs`. For instance, tail
+the rails logs with this.
 ```
-$ docker-compose down
+$ docker-compose logs -f rails
 ```
 
-## Tidying Up
-Docker likes to accumulate cached containers and images, some of which
-are bound to be superfluous and quite large. Here's how to clean those up.
+To spin all the services back down run the following.
 ```
-$ docker container prune -f
-$ docker rmi $(docker images | grep "^<none>" | awk "{print $3}")
+$ docker-compose down
```

Comment:
If you want to destroy volumes (which can be good), you can also do: `docker-compose down -v` to make everything clean.